src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/PairwiseCouplingTrainer.cs

```diff
@@ -218,7 +219,7 @@ public override TTransformer Fit(IDataView input)
     }
 
     /// <summary>
-    /// Contains the model parameters and prediction functions for the PairwiseCouplingTrainer.
+    /// Model parameters for the <see cref="PairwiseCouplingTrainer"/>.
```

Comment:
>the [](start = 29, length = 3)

in some case we are using "the".. in other places (e.g.   GamRegressionModelParameters)  we are not .. it would be good to have consistency

Reply:
used the version without 'the'

---
In reply to: [277840071](https://github.com/dotnet/machinelearning/pull/3516#discussion_r277840071) [](ancestors = 277840071)